mongomock/collection.py

```diff
@@ -1801,9 +1796,23 @@ def _min_updater(doc, field_name, value):
         doc[field_name] = min(doc.get(field_name, value), value)
 
 
+_LAST_TIMESTAMP_INC = collections.defaultdict(int)
```

Comment:
Only the last unix timestamp is needed, not all the used ones (unless we expect going back in time by mocking the `time.time` function).
What do you think about keeping the last used timestamp instead? This might perform better, especially if it is used in an environment with many such updates.

Reply:
Done.